packages/atomic/src/components/search/atomic-results-per-page/atomic-results-per-page.tsx

```diff
@@ -119,22 +120,27 @@ export class AtomicResultsPerPage implements InitializableComponent {
       return;
     }
 
+    const label = this.bindings.i18n.t('results-per-page');
+
     return (
       <div class="flex items-center">
         <span
           part="label"
           class="self-start text-on-background text-lg mr-3 leading-10"
+          aria-hidden="true"
```

Comment:
This label becomes redundant since the `legend` from the `FieldsetGroup` is readable by screen readers.